src/modules/access-wallet/hybrid/handlers/WalletConnect/index.js

```diff
@@ -34,11 +34,12 @@ class WalletConnectWallet {
         store.dispatch('wallet/removeWallet');
       }
     });
-
-    this.walletConnect.disconnect = () => {
-      this.isKilled = true;
-      this.walletConnect.killSession();
-    };
+    if (this.walletConnect) {
+      this.walletConnect.disconnect = () => {
```

Comment:
hmmm the error is `Cannot read property 'disconnect' of null` over here we are setting `.disconnect` property so this is not where `.disconnect()` gets called

Reply:
also the wallettype is web3wallet I'd do a search for `.disconnect()` and check those instances

Reply:
ooohhhhh

Reply:
fixed!